docs/localization.md

```diff
@@ -3,11 +3,20 @@
 This article describes how to localize the Survey Creator UI and give users the ability to localize their surveys.
 
 - [Localize Survey Creator UI](#localize-survey-creator-ui)
+  - [Available Languages](#available-languages)
+  - [Enable Localization and Switch Between Locales](#enable-localization-and-switch-between-locales)
+  - [Override Individual Translations](#override-individual-translations)
 - [Localize Survey Contents](#localize-survey-contents)
 
 ## Localize Survey Creator UI
 
-Survey Creator includes UI translations into several languages. We ship localized strings as [dictionary files](https://github.com/surveyjs/survey-creator/tree/master/packages/survey-creator-core/src/localization). The localization engine that works with these files is available as a separate script/module. Reference this script in the `<head>` tag of your page or import this module in the component that renders Survey Creator:
+### Available Languages
+
+Survey Creator UI is translated into over 30 languages. We ship translated strings as [dictionary files](https://github.com/surveyjs/survey-creator/tree/master/packages/survey-creator-core/src/localization). They are supported by the community and may be incomplete. Feel free to add missing translations to existing dictionaries or create new dictionaries for other languages. You can use English as a base dictionary: copy the file, replace English translations in it, and submit a pull request with the resulting file to the [survey-creator](https://github.com/surveyjs/survey-creator) repository.
```

Comment:
A small correction: there are actually 51 TS localization files in the [localization](https://github.com/surveyjs/survey-creator/tree/master/packages/survey-creator-core/src/localization) repo. Maybe we should mention this like "over 50"?